.github/workflows/ghcr-cleanup.yaml

```diff
@@ -0,0 +1,37 @@
+name: Delete old ghcr images
+on:
+  schedule:
+    - cron: "15 1 * * *"  # every day at 1:15am
+  pull_request:
+    types: [closed]
+
+jobs:
+  pull-request-ghcr-cleanup:
+    if: ${{ github.event_name == 'pull_request' }}
+    name: Delete images related to closed PR
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete images related to closed PR
+        uses: snok/container-retention-policy@v1
+        with:
+          image-names: client-application
+          cut-off: now UTC
+          account-type: org
+          org-name: plgd-dev
+          filter-tags: vnext-pr${{ github.event.pull_request.number }}*
+          token: ${{ secrets.GHCR_CLEANUP_PAT }}
+  nightly-ghcr-cleanup:
+    if: ${{ github.event_name == 'schedule' }}
+    name: Delete old vnext images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete older than a month vnext images
```

Comment:
the cut-off is older than an hour, not a month, right?

Reply:
fixed